api/graphql/application_rounds/application_round_types.py

```diff
@@ -102,12 +101,6 @@ class Meta:
             "target_group",
         ] + get_all_translatable_fields(model)
 
-        filter_fields = {
```

Comment:
Same as above. Do we know that client is not using these filters?

Reply:
Client is using the REST api currently for the application rounds and there are no such filters in use so this should be safe. This was only missing the `pk` filter as was in REST /application_round/<pk>